packages/bls12-381/bls12-381.0.3.14/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "OCaml binding for bls12-381 from librustzcash"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "conf-rust" {build}
+  "ocaml" {>= "4.07"}
+  "dune" {>= "1.7"}
+  "dune-configurator" {build}
+  "ff" {>= "0.4.0" & < "0.5.0"}
+  "zarith" {>= "1.10" & < "2.0"}
+  "ctypes" {= "0.17.1"}
+  "ctypes-foreign" {= "0.4.0"}
```

Comment:
This is a very specific version dependency for ctypes-foreign. Are you sure that >="0.4.0" isnt enough?

Reply:
It should be enough. I should have been less restrictive. It will be lowered in the next release (0.4.0).